frontend/src/components/text-check/IgnoredWordHandler.vue

```diff
@@ -3,22 +3,32 @@ import Button from "primevue/button"
 import { computed } from "vue"
 import { useFeatureToggle } from "@/composables/useFeatureToggle"
 import { Match } from "@/types/textCheck"
+import IconDescription from "~icons/ic/baseline-description"
+import IconSpellCheck from "~icons/material-symbols/spellcheck"
+import IconBookOpenPageVariant from "~icons/mdi/book-open-page-variant"
+import IconBookOpenVariant from "~icons/mdi/book-open-variant"
 
 const props = defineProps<{
   match: Match
 }>()
 
 const emit = defineEmits<{
+  "ignored-word:add": [void]
```

Comment:
Don't we need the word here, that is ignored?
e.g. in TextCheckModal we listen for the emit and take the word to be ignored as parameter. So we need to also emit it?